styles/variables.css

```diff
@@ -60,6 +60,13 @@
     --lightColor-hover:#f7f7f7a2;
     --darkColor-hover:rgb(68, 68, 68);
     --infoColor-hover:#a7dcf5;
+
+    /* Card badges Colors */
+    --redCardBadge:#ed1b24;
+    --orangeCardBadge:#fa7901;
+    --pinkCardBadge:#ee2b8b;
+    --blueCardBadge:#00adee;
+    --greenCardBadge:#b4bd00;
```

Comment:
Same suggestion for css variables

Reply:
I'm looking into this

Reply:
follow this commit I have changed variable names for all files  https://github.com/Rishi05051997/memorable/commit/7a151b1ffcd55f493a8add59be314c9636e026a0